tsdb/engine/tsm1/iterator.go

```diff
@@ -121,3 +121,9 @@ var (
 	nilStringLiteralValueCursor   cursorAt = &literalValueCursor{value: (*string)(nil)}
 	nilBooleanLiteralValueCursor  cursorAt = &literalValueCursor{value: (*bool)(nil)}
 )
+
+func closeAllCursorAt(cur []cursorAt) {
```

Comment:
Could you do something like:

```
type CursorsAt []cursorAt
func (a CursorsAt) Close() {
    for _, c := range a {
        c.close()
    }
}
```

And then `CursorsAt(itr.aux).Close()` or maybe change the type of `aux` and `conds.curs` to `CursorsAt` so you can just call `itr.aux.Close()`?